src/api/apiFetch.js

```diff
@@ -12,11 +13,13 @@ const timeout = (ms) => new Promise(resolve => setTimeout(resolve, ms));
 let activityCounter = 0;
 const activityPush = () => {
   activityCounter++;
-  StatusBar.setNetworkActivityIndicatorVisible(true);
+  if (DeviceInfo.getSystemName() === 'iPhone OS') {
+    StatusBar.setNetworkActivityIndicatorVisible(true);
+  }
 };
 const activityPop = () => {
   activityCounter--;
-  if (activityCounter === 0) {
+  if (activityCounter === 0 && DeviceInfo.getSystemName() === 'iPhone OS') {
```

Comment:
There is no exception but continuous warnings "setNetworkActivityIndicatorVisible is only available on iOS"